vector/v.random/main.c

```diff
@@ -286,7 +288,7 @@ int main(int argc, char *argv[])
             ncols = db_get_table_number_of_columns(table);
             for (icol = 0; icol < ncols; icol++) {
                 col = db_get_table_column(table, icol);
-                sprintf(buf, ",%s_%s %s", parm.input->answer,
+                sprintf(buf, ",%s_%s %s", xname,
```

Comment:
Please change to snprintf() (see #2767) while you’re there. This will be an issue for C code too and a lot of work on GRASS source…
```suggestion
                snprintf(buf, DB_SQL_MAX, ",%s_%s %s", xname,
```